Assets/Scripts/SceneNavigation/EndLevelMenu.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class EndLevelMenu : MonoBehaviour
+{
+    public GameObject endLevelUI;
+
+    void Update()
+    {
+    }
+
+    public void RestartLevel()
+    {
+        Scene scene = SceneManager.GetActiveScene();
+        SceneManager.LoadScene(scene.name);
+        Time.timeScale = 1f;
```

Comment:
NIT: Having this at the bottom of this method and the top of the next method irks me. Leave for now, will sort out in the next PR